rest/src/main/java/com/carhouse/rest/config/SpringJDBCConfig.java

```diff
@@ -58,4 +68,39 @@ public NamedParameterJdbcTemplate getNamedParameterJdbcTemplate(
             final DataSource dataSource) {
         return new NamedParameterJdbcTemplate(dataSource);
     }
+
+    /**
+     * Initialize database.
+     * Used only for tests
+     *
+     * @param dataSource is utility class to access a datasource
+     * @param databasePopulator is class which sets initialization scripts for database
+     * @return DataSourceInitializer
+     */
+    @Bean
+    @Profile("integrationTest")
```

Comment:
Sometimes it's helpful to use enum with all possible profile names and string literals.
Here string literal will be used. 
It helps to avoid typos in profile names

Reply:
Ok, I'll know